host/greatfet/commands/greatfet_led.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+#
+# This file is part of GreatFET
+
+from __future__ import print_function
+
+import errno
+import sys
+import ast
+
+import greatfet
+from greatfet import GreatFET
+from greatfet.utils import log_silent, log_verbose
+from greatfet.peripherals.led import LED
+
+
+def main():
+    from greatfet.utils import GreatFETArgumentParser
+    # Set up a simple argument parser.
+    parser = GreatFETArgumentParser(description="Utility for I2C communication via GreatFET")
```

Comment:
This isn't an I2C tool, I think you need to update the arg parsing code here, as well as the help message.

Reply:
Updated